app/viewModels/UnloadingGuidanceViewModel.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewModels
+
+import javax.inject.Inject
+
+case class UnloadingGuidanceViewModel(title: String,
+                                      heading: String,
+                                      preLinkText: String,
+                                      linkText: String,
+                                      postLinkText: String,
+                                      para1: Option[String],
+                                      para2: String,
+                                      para3: Option[Para3.type]
+) {
+  val prefix = "unloadingGuidance"
+}
+
+case object Para3 {
+  val prefix                    = "unloadingGuidance"
+  val para3preLinkText: String  = s"$prefix.para3.preLinkText"
+  val para3linkText: String     = s"$prefix.para3.linkText"
+  val para3postlinkText: String = s"$prefix.para3.postLinkText"
+}
```

Comment:
Just wondering if this should be a case class, then you might not need "unloadingGuidance" in so many places

```
case class Para3(preLinkText: String, linkText: String, postLinkText: String)
object Para3 {
  def apply(prefix: String): Para3 = new Para3(s"$prefix.para3.preLinkText", s"$prefix.para3.preLinkText", s"$prefix.para3.preLinkText")
}
```